net-im/telegram-desktop-bin/telegram-desktop-bin-4.2.4-r1.ebuild

```diff
@@ -19,6 +19,7 @@ KEYWORDS="-* ~amd64"
 QA_PREBUILT="usr/bin/telegram-desktop"
 
 RDEPEND="
+	!net-im/telegram-desktop
```

Comment:
This is going to require revbumping the ebuild, otherwise this change is meaningless. I'd instead suggest just removing these old versions? :)

And yes a weak blocker is only required in one of the packages, not both. But having it with both doesn't hurt at all either.

Reply:
I see. Thanks for fixing up. The other two will disappear over time, do not want to remove them because of that minor change and risking dropping a fallback option in case that new had any issues.